src/libraries/System.Text.Json/tests/System.Text.Json.Tests/Serialization/JsonElementTests.cs

```diff
@@ -219,22 +220,35 @@ public static void DeepEquals_NotEqualValuesReturnFalse(string value1, string va
         }
 
         [Theory]
-        [InlineData(10)]
-        [InlineData(100)]
-        [InlineData(500)]
```

Comment:
Is the depth of 500 still crashing in some platforms? I'm surprised this didn't turn up in CI when I had the PR up.

Reply:
Yes, it failed locally for me.